### PR TITLE
Add option to Render the heading#1 for letting readthedocs to generate TOC automatically #214

### PR DESCRIPTION
**Pull request by t6847kimo**
Hi, sorry for late response.
Please take a look at it, thanks!

### CODE REVIEW

File: src/pydoc_markdown/contrib/renderers/markdown.py [hunk 1]
```diff
@@ -373,12 +376,19 @@ def render_to_string(self, modules: t.List[docspec.Module]) -> str:
     self.render_to_stream(modules, fp)
     return fp.getvalue()
 
-  def render_to_stream(self, modules: t.List[docspec.Module], stream: t.TextIO):
+  def render_to_stream(self, modules: t.List[docspec.Module], stream: t.TextIO, page_title:str="API documentation"):
```
Comment (NiklasRosenstein) [inline src/pydoc_markdown/contrib/renderers/markdown.py]:
```suggestion
  def render_to_stream(self, modules: t.List[docspec.Module], stream: t.TextIO, page_title: str = "API documentation"):
```

nit

File: src/pydoc_markdown/contrib/renderers/markdown.py [hunk 2]
```diff
@@ -394,12 +404,12 @@ def get_resolver(self, modules: t.List[docspec.Module]) -> t.Optional[Resolver]:
 
     return MarkdownReferenceResolver(modules)
 
-  def render(self, modules: t.List[docspec.Module]) -> None:
+  def render(self, modules: t.List[docspec.Module], page_title:str) -> None:
```
Comment (NiklasRosenstein) [inline src/pydoc_markdown/contrib/renderers/markdown.py]:
```suggestion
  def render(self, modules: t.List[docspec.Module], page_title: str) -> None:
```
nit

**Review by NiklasRosenstein — CHANGES_REQUESTED** [review]
Hey @t6847kimo , thanks for the PR. I have to think about this a bit more. The `Renderer.render()` method does not accept a `page_title` argument and I'm unsure if I want to add it to the interface.

**Comment by NiklasRosenstein** [comment]
I just noticed that I even [suggested this](https://github.com/NiklasRosenstein/pydoc-markdown/issues/214#issuecomment-912430831) 🥲 

> There's many was this could be done. I think I would prefer an option in the Mkdocs renderer like render_page_title and that option is either passed to Page.render() or is set as an option in the Markdown renderer along with the title string.

**Comment by t6847kimo** [comment]
Hi, may I know the current status?
Can I help anything?

**Comment by NiklasRosenstein** [comment]
Hey @t6847kimo , sorry for the long silence. I've made a few changes before merging, that's why I create a new PR for it. Thanks for the contribution!

**Pull request closed without merging**